goaviatrix/account.go

```diff
@@ -166,6 +187,98 @@ func (c *Client) CreateOCIAccount(account *Account) error {
 	return nil
 }
 
+func (c *Client) CreateAWSC2SAccount(account *Account) error {
+	params := map[string]string{
+		"CID":                       c.CID,
+		"action":                    "setup_account_profile",
+		"account_name":              account.AccountName,
+		"cloud_type":                strconv.Itoa(account.CloudType),
+		"aws_orange_account_number": account.AwsOrangeAccountNumber,
+		"aws_orange_cap_url":        account.AwsOrangeCapUrl,
+		"aws_orange_cap_agency":     account.AwsOrangeCapAgency,
+		"aws_orange_cap_mission":    account.AwsOrangeCapMission,
+		"aws_orange_cap_role_name":  account.AwsOrangeCapRoleName,
+	}
+
+	files := []File{
+		{
+			Path:      account.AwsOrangeCapCert,
+			ParamName: "aws_orange_cap_cert",
+		},
+		{
+			Path:      account.AwsOrangeCapCertKey,
+			ParamName: "aws_orange_cap_cert_key",
+		},
+		{
+			Path:      account.AwsOrangeCaChainCert,
+			ParamName: "aws_orange_ca_chain_cert",
+		},
+	}
+
+	resp, err := c.PostFile(c.baseURL, params, files)
```

Comment:
Can use PostFileAPI function for all of these to avoid repeating the response decoding logic